frontend/src/metabase/query_builder/components/expressions/Editor/HelpText/HelpText.module.css

```diff
@@ -0,0 +1,79 @@
+.usage {
+  padding-left: 0.5rem;
+  background-color: white;
+  color: var(--mb-color-text-secondary);
+  cursor: pointer;
+  font-family: var(--mb-default-monospace-font-family);
+  flex-shrink: 0;
+  font-size: 0.75rem;
+  min-height: 2.5rem;
+  line-height: 2.5rem;
+  box-sizing: border-box;
+
+  & + ul {
+    height: calc(100% - 5rem);
+  }
+}
+
+.toggle {
+  margin-left: auto;
+  padding: 0 0.65rem;
+}
+
+.chevron {
+  color: var(--mb-color-text-secondary);
+  padding-top: 1px;
+
+  &.open {
+    transform: rotate(90deg);
+  }
+}
+
+.arg {
+  color: var(--mb-base-color-dubloon-50);
+  font-family: var(--mb-default-monospace-font-family);
+  font-size: 0.75rem;
```

Comment:
Text in left and right column is misaligned vertically

![image](https://github.com/user-attachments/assets/d0b118a7-c3b0-4aff-829a-24f8042f6b5d)